tough/tests/http.rs

```diff
@@ -1,7 +1,11 @@
 mod test_utils;
 
 /// Instead of guarding every individual thing with `#[cfg(feature = "http")]`, use a module.
```

Comment:
I guess this comment also needs modification.

Reply:
This was never a very useful comment so I deleted it.